modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/filter/expression/PredicateExpressionVisitorImpl.java

```diff
@@ -740,6 +750,38 @@ private Object _getValue(
 		}
 	}
 
+	private Object _handleComplexPropertyExpressionPropertyExpression(
+		PropertyExpression propertyExpression, List<String> acc) {
```

Comment:
I don't understand the meaning of `acc`

Reply:
Accumulator 😁 😁 😁 
I will try to set a more descriptive name

Reply:
Done